src/containerapp/azext_containerapp/_params.py

```diff
@@ -435,7 +435,7 @@ def load_arguments(self, _):
     with self.argument_context('containerapp session code-interpreter', arg_group='file') as c:
         c.argument('filename', help="The file to delete or show from the session")
         c.argument('filepath', help="The local path to the file to upload to the session")
-        c.argument('path', help="The path to list files from the session")
+        c.argument('path', help="The path of the file from the session")
```

Comment:
How about `The path of files in the session`. `from`/`to` terms are used w/ verbs, like above `upload to` or `delete from`.

Reply:
Thanks, fixed